styles/ContactPage.module.css

```diff
@@ -0,0 +1,126 @@
+.contact {
+  width: 100%;
+}
+.msg__container {
+  display: flex;
+  align-items: center;
+  height: 500px;
+  border-bottom: 1px solid var(--border-default);
+}
+.msg_left__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-right: 10%;
+  width: 50%;
+  height: 80%;
+}
+.msg_left_title__container {
+  width: 100%;
+  height: 50%;
+}
+.msg_left_title__heading {
+  margin-bottom: 2%;
+}
+.social_media__container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+.social {
+  margin-bottom: 2%;
+}
+.msg_form__container {
+  width: 35%;
+  height: 70%;
+}
+.label_input__container {
+  position: relative;
+}
+.msg__input {
+  width: 100%;
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-width: 1px;
```

Comment:
May be we can use border-bottom here?

Reply:
I tried, but the default value of the bottom has these styles, and I think it is necessary to set them to none.